addons/stock_picking_batch/views/stock_picking_batch_views.xml

```diff
@@ -68,13 +68,21 @@
         <field name="arch" type="xml">
             <form string="Stock Batch Transfer">
                 <field name="show_check_availability" invisible="1"/>
+                <field name="show_set_all_done" invisible="1"/>
                 <header>
                     <button name="action_confirm" states="draft" string="Confirm" type="object" class="oe_highlight"/>
+                    <button name="action_set_done_quantities" string="Set All Done" type="object" class="oe_highlight"
+                        attrs="{'invisible': [
+                            '|',
+                                ('state', '!=', 'in_progress'),
+                                ('show_set_all_done', '=', False)]}"/>
                     <button name="action_done" string="Validate" type="object" class="oe_highlight"
                         attrs="{'invisible': [
                             '|',
                                 ('state', '!=', 'in_progress'),
-                                ('show_check_availability', '=', True)]}"/>
+                                '|',
+                                    ('show_check_availability', '=', True),
+                                    ('show_set_all_done', '=', True)]}"/>
```

Comment:
You should be able to view both button at the same time in batch picking. You could validate all the picking already processed without forcing the quantity done on the other